tests/test-cases/Group23-VIC-Machine-Service/TestCases.md

```diff
@@ -0,0 +1,26 @@
+Group 23 - VIC Machine Service
```

Comment:
Should this new test group be added to `tests/test-cases/TestGroups.md`?

Reply:
Yep, good catch! 👍